RollOfHonour/src/RollOfHonour.Web/appsettings.json

```diff
@@ -1,9 +1,32 @@
 {
+  /*
+The following identity settings need to be configured before the project can be successfully executed. For more info see https://aka.ms/dotnet-template-ms-identity-platform 
+*/
+  "AzureADB2C": {
+    "Instance": "https://--your-domain--.b2clogin.com",
+    "Domain": "[Enter the domain of your B2C tenant, e.g. contoso.onmicrosoft.com]",
+    "TenantId": "[Enter 'common', or 'organizations' or the Tenant Id (Obtained from the Azure portal. Select 'Endpoints' from the 'App registrations' blade and use the GUID in any of the URLs), e.g. da41245a5-11b3-996c-00a8-4d99re19f292]",
+    "ClientId": "[Enter the Client Id (Application ID obtained from the Azure portal), e.g. ba74781c2-53c2-442a-97c2-3d60re42f403]",
+    // Use either a secret or a certificate. ClientCertificates are recommended.
+    "ClientSecret": "[Copy the client secret added to the app from the Azure portal]",
+    //"ClientCertificates": [
+    //],
+    // the following is required to handle Continuous Access Evaluation challenges
+    "ClientCapabilities": [ "cp1" ],
+    "CallbackPath": "/signin-oidc",
```

Comment:
Have all these settings been configured in the app config instance? Feels weird to add this config in here if we plan to drive it from Azure

Reply:
Yeah they are all in the Azure app config too.  This is helpful when grabbing json to steal to se in secrets locally when in dev.